css/index.css

```diff
@@ -112,6 +129,46 @@ p {
     justify-content: center;
 }
 
```

Comment:
Looks good, you could also add a second class in stead of adding an ID on the boxes, but that's really a preference kinda thing, not necessary. This way is perfectly acceptable too